vispy/visuals/implicitmesh.py

```diff
@@ -0,0 +1,66 @@
+
+from .mesh import MeshVisual
+from ..geometry import create_implicit_mesh
+from ..color import ColorArray
+
+import numpy as np
+
+
+class ImplicitMeshVisual(MeshVisual):
+    """Displays a mesh implicitly constructed around x,y,z coordinates.
+
+    This makes it simple to generate a mesh from e.g. the output
+    of numpy.meshgrid.
+
+    Parameters
+    ----------
+    xs : ndarray
+        A 2d array of x coordinates for the vertices of the mesh. Must
+        have the same dimensions as ys and zs.
+    ys : ndarray
+        A 2d array of y coordinates for the vertices of the mesh. Must
+        have the same dimensions as xs and zs.
+    zs : ndarray
+        A 2d array of z coordinates for the vertices of the mesh. Must
+        have the same dimensions as xs and ys.
+    color : Color | ColorArray | ndarray
+        The color(s) of the points of the mesh. If a Color or ColorArray
+        is passed, the vertex_colors are constructed by cycling their
+        contents. If an ndarray of ndim 3 is passed, it is assumed that
+        this has the same shape as the xs, ys and zs arrays and gives
+        one colour per vertex.
+    shading : str | None
+        Same as for the `MeshVisual` class. Defaults to 'smooth'.
+    vertex_colors: ndarray | None
+        Same as for the `MeshVisual` class. Defaults to None. Overrides
+        the color argument if set.
+    face_colors: ndarray | None
+        Same as for the `MeshVisual` class. Defaults to None.
+    mode : str
```

Comment:
This is not needed--the visual generates the mesh indices and therefore knows exactly what triangle mode should be used.